<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HorizontalLinearStageWidget</class>
 <widget class="QWidget" name="HorizontalLinearStageWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>745</width>
    <height>225</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>HorizontalLinearStageWidget</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_12">
     <item>
      <layout class="QFormLayout" name="formLayout_4">
       <item row="0" column="0">
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Horizontal Axis Device</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <layout class="QHBoxLayout" name="horizontalLayout_8">
         <item>
          <widget class="QComboBox" name="DeviceComboBox"/>
         </item>
         <item>
          <widget class="QPushButton" name="DeviceSetBtn">
           <property name="text">
            <string>Connect</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Acceleration</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Speed</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Position Step</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QDoubleSpinBox" name="SpeedSpinBox">
         <property name="maximum">
          <double>50000.000000000000000</double>
         </property>
         <property name="value">
          <double>10000.000000000000000</double>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QDoubleSpinBox" name="AccelSpinBox">
         <property name="maximum">
          <double>50000.000000000000000</double>
         </property>
         <property name="value">
          <double>10000.000000000000000</double>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QDoubleSpinBox" name="PosStepSpinBox">
         <property name="maximum">
          <double>10000.000000000000000</double>
         </property>
         <property name="value">
          <double>1000.000000000000000</double>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="YAxisControl">
     <item>
      <widget class="QPushButton" name="YLeftMoveBtn">
       <property name="text">
        <string>&lt;</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="YStopBtn">
       <property name="text">
        <string>stop</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="YRightMoveBtn">
       <property name="text">
        <string>&gt;</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
